unstract/task-abstraction/tests/run_tests.py

```
@@ -0,0 +1,206 @@
+
+
+def test_full_workflow():
+    """Test complete workflow."""
+    from task_abstraction import get_backend
+    from task_abstraction.models import BackendConfig
+
+    # Test with default config (no external dependencies)
+    try:
+        # This will fail because Celery isn't installed, but error handling should work
+        get_backend("celery", use_env=False)
+        assert False, "Should have failed due to missing Celery"
```

Ruff (0.13.1)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)